### ISSUE TITLE
Crash when breaking smeltery wall - pack v4.0.2

### ISSUE THREAD

**Issue by ArbitraryHubris** [Darkosto/SkyFactory4Issues#580]
[Crash Report](https://pastebin.com/qAgDNgH0)

This occurred in v4.0.2 of the pack.  There is one difference in our server's set of mods.  I upgraded ComPatchedStorage to v1.5.1 to alleviate the visual glitching when viewing inventory in compact storage chests.

The user who initiated the crash was breaking a block in the wall of the smeltery, which brought the server down.  I asked if he had any knightslime-related alloys in the smeltery and he said he didn't.  He did say, though, that his smeltery was full to the limit.

My apologies if I didn't include something you need - for some reason I was not able to get the issue template to show up.

**Comment by sk2048** [comment]
Slimeknights is just the name of the team who make Tinkers', but you could recreate this crash with knightslime if you really wanted to :P

I've managed to replicate this, and have submitted a report to the mod.

SlimeKnights/TinkersConstruct#3836

**Comment by ArbitraryHubris** [comment]
Yeah, I figured it was a red herring, but I look for breadcrumbs and talk with server members to find what was different.  When you recreated it, was a full smeltery the key? @sk2048

**Comment by sk2048** [comment]
It had to be a full smeltery that was actively making an alloy.
